dlp/README.md

```diff
@@ -33,11 +33,7 @@ These samples show how to use the [Google Cloud DLP API](https://cloud.google.co
 
 1. **Run samples**
     ```
-    Usage: ruby sample.rb <command> [arguments]
-
-    Commands:
-      inspect_string <content> <max_findings> Inspect a string.
-      inspect_file <filename> <max_findings> Inspect a local file.
+    Usage: ruby filename.rb [arguments]
```

Comment:
I think having examples of how to run each sample file could be helpful if it can be called through the CLI. Without instructions I wouldn't know how to.

Reply:
`+1`

For **hand-written** samples, we should maybe:
 - Minimal to no CLI (minimal may be preferable for testing)
 - No CLI documentation
 - No README required either?

For **generated** samples, `+1` for having some more specific `Usage:` documentation (for each command, like we've historically been authoring by hand)

Reply:
Listed commands in https://github.com/GoogleCloudPlatform/ruby-docs-samples/pull/356/commits/2ef920853952b386e644be14bebec3e2471b7edf